### PR TITLE
Affinity for processors pipeline. Fix integration tests.

### PR DESCRIPTION
**Pull request by KochetovNicolai**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Non-significant (changelog entry is not needed)

Changelog entry (up to few sentences, required except for Non-significant/Documentation categories):

### CODE REVIEW

**Comment by azat** [comment]
FWIW with `experimental_use_processors` (v19.19.1.1929-testing-163-g8711f98c30) I got `Pipeline stuck` error from time to time, I will try to come back with minimal reproducer (or maybe this is not an issues, and caused by for example network errors or similar)

**Comment by KochetovNicolai** [comment]
@azat cool! If you have a reproducible example which fails at least sometimes I would be happy to see it.

File: dbms/src/Processors/Executors/PipelineExecutor.cpp [hunk 1]
```diff
@@ -11,6 +11,8 @@
 #include <Processors/ISource.h>
 #include <Common/setThreadName.h>
 
+#include <sched.h>
```
Comment (azat) [inline dbms/src/Processors/Executors/PipelineExecutor.cpp]: Looks like this also can be omitted for apple/freebsd

Comment (KochetovNicolai) [reply dbms/src/Processors/Executors/PipelineExecutor.cpp]: Yes.
Actually, it's better to add affinity for other os too.

**Pull request merged**